varats-core/varats/report/gnu_time_report.py

```diff
@@ -197,3 +200,38 @@ def _parse_max_resident_size(line: str) -> int:
         raise WrongTimeReportFormat(
             "Could not parse max resident set size: ", line
         )
+
+
+class TimeReportAggregate(
+    ReportAggregate[TimeReport],
+    shorthand=TimeReport.SHORTHAND + ReportAggregate.SHORTHAND,
+    file_type=ReportAggregate.FILE_TYPE
+):
+    """Context Manager for parsing multiple time reports stored inside a zip
+    file."""
+
+    def __init__(self, path: Path) -> None:
+        super().__init__(path, TimeReport)
+
+    @property
+    def wall_clock_times(self) -> tp.List[float]:
+
+        return [
+            report.wall_clock_time.total_seconds() for report in self.reports
+        ]
+
+    @property
+    def wall_clock_time_mean(self) -> float:
```

Comment:
```suggestion
    def mean_wall_clock_time(self) -> float:
```
I think this would read better.

```
df = ...
report = ...
df["mean"] = report.mean_wall_clock_time
```